### DISCUSSION TITLE
If i have product VPS and have attribute Ram, 1 ram price 10$, 10 ram = 100$. I must will create 10 product variant?

### DISCUSSION THREAD

**Discussion by firer777** [lunarphp/lunar#1946] [Q&A]
I first time use this package. Please help me this question on title

**Comment by alecritson** [comment]
From what I understand it could be one product with one variant and you have a base price of $10, you could then use quantity to determine the amount of ram and adjust the price.

**Reply by firer777** [reply]
Yeb, I want create a product name example: VPS 01 price 20$ and this product have more addons like Ram,CPU. I want define price each addon example Ram: 1 GB - price 10$, CPU: 1 cores - price 5$, if customer want buy 5 GB Ram and 2 CPU cores. Lunar CMS can do it or must create more product variant  and define price for each options example, 1GB: 10$ + base price VPS, 2GB: 20$ + base price VPS,.... ? Sr my english
